addons/hr_attendance/static/src/components/attendance_menu/attendance_menu.js

```diff
@@ -25,12 +26,22 @@ export class ActivityMenu extends Component {
         this.dropdown = useDropdownState();
         // load data but do not wait for it to render to prevent from delaying
         // the whole webclient
-        this.searchReadEmployee();
+        onWillStart(()=> {
+            this.lazySession.getValue("attendance_user_data").then((employee) => {
```

Comment:
// access lazy session but do no wait for it, to prevent from delaying the whole webclient